packages/micro-analytics-cli/src/handler.js

```diff
@@ -24,6 +24,13 @@ function realtimeHandler(req, res) {
   }
 }
 
+async function readMeta(req) {
+  if (await text(req)) {
+    return (await json(req)).meta;
```

Comment:
I think it'd be better to silently fail since only the owner of the service should really be aware that something went wrong? I wouldn't want random users POST'ing stuff to get some errors back.